logging/log.go

```diff
@@ -0,0 +1,82 @@
+package logging
+
+import (
+	"time"
+
+	grpc_logging "github.com/grpc-ecosystem/go-grpc-middleware/logging"
+	grpc_logrus "github.com/grpc-ecosystem/go-grpc-middleware/logging/logrus"
+	"github.com/sirupsen/logrus"
+)
+
+func NewLogger(level string) *logrus.Logger {
```

Comment:
Should we instead of using simple string define custom type 
```go
type logLevel logrus.Level
```

and define a few public constansts 
```go
const (
   DebugLevel = logrus.DebugLevel
   InfoLevel = logrus.InfoLevel
   ...
)
```

This helps us to make this function safe and forbide clients pass bad level

Reply:
I think it's better to leave this argument as is and use already written levels from logrus. Don't see reasons to rewrite them